src/search/searchBar.component.test.tsx

```diff
@@ -61,11 +61,20 @@ describe('searchBar component', () => {
 
     const dateFilterFromDate = screen.getByLabelText('from, date-time input');
     const dateFilterToDate = screen.getByLabelText('to, date-time input');
-    await user.clear(dateFilterFromDate);
-    await user.clear(dateFilterToDate);
 
-    await user.type(dateFilterFromDate, '2022-01-01 00:00');
-    await user.type(dateFilterToDate, '2022-01-02 00:00');
+    // .clear doesn't work with datepickers in v6
+    await user.click(dateFilterFromDate);
+    await user.keyboard('{Control>}a{/Control}');
+    await user.keyboard('{Delete}');
+    // await user.clear(dateFilterFromDate);
+
+    await user.click(dateFilterToDate);
+    await user.keyboard('{Control>}a{/Control}');
+    await user.keyboard('{Delete}');
+    // await user.clear(dateFilterToDate);
```

Comment:
We can use `clear` in unit tests now

Reply:
I checked and it still doesn't seem to be working 🤔

Reply:
`@mui/x-date-pickers` needs to be upgraded to `6.16.0`

Reply:
I'm pushing a commit that updates the package and our patch

Reply:
This might mean we can use the `clear` command in e2e tests now as well

Reply:
In fact, the update broke the e2e tests even more